db/migrate/20221122174527_create_post.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] Kindly add the `comments_counter` of integer type as well as the `likes_counter` field of integer type as rhis is part of the requirements